gocd/checkers.suse.gocd.yaml.erb

```diff
@@ -281,6 +281,35 @@ pipelines:
             - staging-bot
             tasks:
             - script: ./legal-auto.py -A https://api.suse.de --debug --legaldb http://legaldb.suse.de project $(cat /home/go/config/legal-auto-projects-suse)
```

Comment:
please program this as loop then

Reply:
sorry, but did you mean by "loop"

Reply:
Well, replace the copy&pasted code with a loop over SP1 and SP2. this is ruby

Reply:
Take inspiration from 

https://github.com/openSUSE/openSUSE-release-tools/blob/6956f3b65580b41069282186c99ae254aa83bfe7/gocd/announcer.gocd.yaml.erb#L3